### PR TITLE
[7.0] Add Scheduler Error Message Post

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by sbidoul** [comment]
@gurneyalex do you know why the runbot builds are marked "skipped"?

**Comment by moylop260** [comment]
@sbidoul I'll review it

**Comment by moylop260** [comment]
> @sbidoul  do you know why the runbot builds are marked "skipped"?

This is because we don't have a explicit build with `TESTS="1"` in our [.travis.yml#7.0](https://github.com/OCA/server-tools/blob/7.0/.travis.yml)

```bash
env:
  - VERSION="7.0" LINT_CHECK=1
  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK=0
  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK=0
  - VERSION="7.0" UNIT_TEST="1" LINT_CHECK=0 EXCLUDE=record_archiver
```

The following patch could fix it
```diff
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK=0
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK=0 TESTS="1"
-  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK=0
+  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK=0 TESTS="1"
```

Or a migration to new style `global-matrix` is better:
 - [sample_files/.travis.yml](https://github.com/OCA/maintainer-quality-tools/blob/d63559c71754a365a0d63b4dcaec528978c33da5/sample_files/.travis.yml#L33-L64)

File: .travis.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ python:
 
 env:
   - VERSION="7.0" LINT_CHECK=1
-  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK=0
-  - VERSION="7.0" ODOO_REPO="OCA/OCB" LINT_CHECK=0
+  - VERSION="7.0" ODOO_REPO="odoo/odoo" LINT_CHECK=0 TESTS="1"
```
Comment (moylop260) [inline .travis.yml:17]: 👍 Now runbot is working!

**Pull request closed without merging**